extensions/BugModal/web/bug_modal.js

```diff
@@ -868,12 +868,23 @@ $(function() {
             if ($('#comment').val() != reply_text) {
                 $('#comment').val($('#comment').val() + reply_text);
             }
-            autosize.update($('#comment'));
+            if (BUGZILLA.user.settings.autosize_comment) {
+                autosize.update($('#comment'));
+            }
+        })
+
+            // calling autosize.update or autosize
+            if (BUGZILLA.user.settings.autosize_comments) {
+                autosize($('#comment'));
+            }
             $.scrollTo($('#comment'), function() { $('#comment').focus(); });
-        });
 
-    // auto-enlarge comment area (up to its max-height)
-    autosize($('#comment'));
+            else if (BUGZILLA.user.settings.zoom_textareas) {
```

Comment:
this else belongs after the }, and it isn't valid to have code between } and an else.